pkg/defaults/defaults.go

```diff
@@ -42,6 +42,9 @@ const (
 
 	// Default secure export logs permissions
 	DefaultLogsPermission = "600"
+
+	// Pid file where to write tetragon main PID
+	DefaultPidFile = DefaultRunDir + "tetragon.pid"
```

Comment:
nit: I would use `path.join` in case the `/` gets removed someday or whatever.

Reply:
agree, but don't want to import extra packages there, think it is fine, thank you ;-)